docs/index.njk

```diff
@@ -205,7 +205,7 @@ title: Home
           <div class="col-10 col-md-6 col-lg-5 offset-lg-1">
               <p class="feature-big-text">Start using Pelican now.</p>
           </div>
-          <div class="col-10 col-md-6 col-lg-5 text-right">
+          <div class="col-10 col-md-6 col-lg-5 text-center text-lg-right">
```

Comment:
Not sure if this makes much sense from what's in figma; currently at medium size the elements appear side-by-side, so why is the button centered then and then to the right at large?
Also, pretty sure the text above should get the same treatment, considering this is what it appears as:
![image](https://user-images.githubusercontent.com/6549759/155574431-156e266a-3950-4837-adc8-d5d293032ce8.png)

Reply:
Good catch. thx